File: api-server/src/main/java/com/talentlink/talentlink/chat/dto/ChatRoomListItemDto.java
package com.talentlink.talentlink.chat.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class ChatRoomListItemDto {

    private Long roomId;
    private String opponentNickname;
    private String opponentProfileImageUrl; // 필드 이름을 opponentProfileUrl -> opponentProfileImageUrl 로 수정
    private String lastMessage;
    private LocalDateTime lastMessageAt;
    private Long unreadCount;

}

File: front-server/app/src/main/java/com/example/talent_link/ui/chat/dto/ChatRoomListItemDto.kt
package com.example.talent_link.Chat.dto

data class ChatRoomListItemDto(
    val roomId: Long,
    val opponentNickname: String,
    val opponentProfileImageUrl: String?,
    val lastMessage: String,
    val lastMessageAt: String?,
    val unreadCount: Long
)
